src/components/eachBook.js

```diff
@@ -1,31 +1,74 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// import './styles/Book.css';
 import { useDispatch } from 'react-redux';
-import { deleteBookFetch} from '../redux/Books/Books';
+// import { Progress } from 'antd';
+import { removeBook } from '../redux/Books/Books';
 
-const  ListOfBooks = ({ item_id, title, author }) => {
-  
+const Book = ({ book }) => {
   const dispatch = useDispatch();
 
-  const removeHandler = (e) =>{
-    const targetId = Number(e.target.id);
-    dispatch(deleteBookFetch(targetId));
+  const handleRemove = (id) => {
+    dispatch(removeBook(id));
   };
 
   return (
-    <li key={item_id}>
-      {title}
-      {' '}
-      by
-      {' '}
-      {author}
-      {' '}
-      <button type="button" id={item_id} className='remove button'  onClick={(e) => removeHandler(e)}>Remove</button>
-    </li>
+    <>
+      <li key={book.item_id} className="bookList">
+        <div className="bookItem">
+          <p className="BookCategory">{book.category}</p>
+          <h3>{book.title}</h3>
+          <small>{book.author}</small>
+          <div className="btn-collection">
+            <button
+              type="button"
+              className="comment-btn"
+            >
+              Comments
+            </button>
+            <button
+              onClick={() => handleRemove(book.item_id)}
+              type="button"
+              className="delete-btn"
+            >
+              Remove
+            </button>
+            <button
+              type="button"
+              className="edit-btn"
+            >
+              Edit
+            </button>
+          </div>
+        </div>
+        <div className="completed">
+          <span className="oval circular-chart">
+            {/* <Progress
+              type="circle"
+              width={80}
+              format={() => ''}
+              strokeColor={{
+                '0%': '#307bbe',
+                '100%': '#379cf6',
+              }}
+              trailColor="#e8e8e8"
+              percent={book.completed}
+            /> */}
```

Comment:
OPTIONAL
_I make it optional but this is a general requirement for every project._
- [ ] To make clean code kindly delete commented code.